<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileRotationWindow</class>
 <widget class="QMainWindow" name="TileRotationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="BuildingEditor::FurnitureView" name="furnitureView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="tilesetLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="tilesetFilter"/>
          </item>
          <item>
           <widget class="QListWidget" name="tilesetList"/>
          </item>
          <item>
           <widget class="QToolButton" name="tilesetMgr">
            <property name="text">
             <string>Tilesets...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Tiled::Internal::MixedTilesetView" name="tilesetTilesView">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionAddTiles"/>
   <addaction name="actionClearTiles"/>
   <addaction name="actionRemove"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-open.png</normaloff>:/images/16x16/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-save.png</normaloff>:/images/16x16/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-save-as.png</normaloff>:/images/16x16/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/document-new.png</normaloff>:/images/16x16/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/edit-delete.png</normaloff>:/images/16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionAddTiles">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/22x22/add.png</normaloff>:/images/22x22/add.png</iconset>
   </property>
   <property name="text">
    <string>Add Tiles</string>
   </property>
  </action>
  <action name="actionClearTiles">
   <property name="icon">
    <iconset resource="tiled.qrc">
     <normaloff>:/images/16x16/edit-clear.png</normaloff>:/images/16x16/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Tiles</string>
   </property>
   <property name="toolTip">
    <string>Clear selected tiles</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::Internal::MixedTilesetView</class>
   <extends>QTableWidget</extends>
   <header>BuildingEditor/mixedtilesetview.h</header>
  </customwidget>
  <customwidget>
   <class>BuildingEditor::FurnitureView</class>
   <extends>QTableWidget</extends>
   <header>BuildingEditor/furnitureview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tiled.qrc"/>
 </resources>
 <connections/>
</ui>
